desc: Avoid misleading array limit error message
table_variable_name: tbl
tests:
  - py: tbl.insert([{'id':i,'mod':i%5,'foo':5} for i in range(1,1000)]).pluck('first_error', 'inserted')
    ot: ({'inserted':999})
  - py: tbl.coerce_to('array')
    runopts:
      array_limit: 500
    ot: err("RqlRuntimeError", "Array over size limit `500`.", [0])
  - py: tbl.group('mod').coerce_to('array')
    runopts:
      array_limit: 500
    ot: err("RqlRuntimeError", "Grouped data over size limit `500`.  Try putting a reduction (like `.reduce` or `.count`) on the end.", [0])
  - py: tbl.group('foo').coerce_to('array')
    runopts:
      array_limit: 500
    ot: err("RqlRuntimeError", "Grouped data over size limit `500`.  Try putting a reduction (like `.reduce` or `.count`) on the end.", [0])
